inception-ui-workload-management-and-monitoring/src/main/java/de/tudarmstadt/ukp/inception/app/ui/monitoring/page/Monitoring.java

```diff
@@ -0,0 +1,166 @@
+    public static final String INPROGRESS = "In Progress";
+    public static final String FINISHED = "Finished";
+
+    //SpringBeans
+    private @SpringBean UserDao userRepository;
+    private @SpringBean ProjectService projectService;
+    private @SpringBean MenuItemRegistry menuItemService;
+    private @SpringBean DocumentService documentService;
+
+    //Constructor
+    public Monitoring(final PageParameters aPageParameters)
+    {
+        //Get current user
+        User user = userRepository.getCurrentUser();
+
+        //Get current Project
+        StringValue projectParameter = aPageParameters.get(PAGE_PARAM_PROJECT_ID);
+        Optional<Project> currentProject = getProjectFromParameters(projectParameter);
+
+        //get Documents for current project
+        List<SourceDocument> documentList = documentService.listSourceDocuments(currentProject.
+            get());
+
+        //Headers of the Table
+        List<String> headers = new ArrayList<>();
+        headers.add(DOCUMENT);
+        headers.add(INPROGRESS);
+        headers.add(FINISHED);
+
+        //Create empty List with data
+        //Create provider with empty list
+        List<List<String>> data = new ArrayList<>();
+        for (int i = 0; i < documentList.size(); i++)
+        {
+            data.add(new ArrayList<>());
+            for (int j = 0; j < headers.size(); j++)
+            {
+                data.get(i).add(j,"");
+            }
+        }
+
+        //Data Provider for the able
+        DataProvider dataProvider = new DataProvider(headers, data);
+
+        //Columns of the table
+        List<IColumn<?,?>> columns = new ArrayList<>();
+        for (int j = 0; j < dataProvider.getTableHeaders().size(); j++)
+        {
+            //Each column creates TableMetaData
+            columns.add(new TableMetaData(dataProvider, j, documentList));
+        }
+
+        //The DefaultDataTable
+        table = new DefaultDataTable("dataTable", columns, dataProvider, 10);
+
+        //Miscellaneous for fields
+        //DEFAULTDOCUMENTSTEXTFIELD.setRequired(true);
+
+
+        //Add to page according to the following structure
+
+        //Menu first
+        menu = new DashboardMenu("menu", LoadableDetachableModel.of(this::getMenuItems));
+        add(menu);
+
+        //Then default annotations
+        add(DEFAULTDOCUMENTSTEXTFIELD);
+
+        //Finally table
+        add(table);
+
+    }
+
+
+    //Return current project
+    private Optional<Project> getProjectFromParameters(StringValue aProject)
+    {
+        if (aProject == null || aProject.isEmpty())
+        {
+            return Optional.empty();
+        }
+
+        try
+        {
+            return Optional.of(projectService.getProject(aProject.toLong()));
+        }
+        catch (NoResultException e)
+        {
+            return Optional.empty();
+        }
+    }
+
+    //Used for creating the menu
+    private List<MenuItem> getMenuItems()
```

Comment:
Please don't add the left handside menu from the dashboard to the page to stay consistent with other pages. You might want to test that you can use the back button/ dashboard link to navigate back to the dashboard project view.